src/jsSSE.php

```diff
@@ -88,7 +91,11 @@ public function flush()
      */
     private function output($content)
     {
-        echo $content;
+        if ($this->echoFunction) {
```

Comment:
i think using `if (is_callable($this->echoFunction))` here would be better / more correct

Reply:
this has hidden danger. If you specify a non-existent callback, nothing will happen and you will bang your head against the table just because of the typo or private method.